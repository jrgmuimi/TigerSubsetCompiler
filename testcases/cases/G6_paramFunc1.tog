let
    function Pfunc(p1:int):void = 
        printstring("I got a param! - ");
        printint(p1)
    end
in
    Pfunc(123456789)
end