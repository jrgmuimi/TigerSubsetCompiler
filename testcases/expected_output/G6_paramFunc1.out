MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar

I got a param! - 123456789
